test/unit/double_free.c

```diff
@@ -101,10 +103,38 @@ TEST_BEGIN(test_small_double_free_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_small_double_free_arena) {
+	test_skip_if(!config_debug);
+	test_skip_if(opt_debug_double_free_max_scan == 0);
+	test_skip_if(!tcache_enabled());
+
+	test_double_free_pre();
+	/* Allocate one more pointer to avoid empty cache after flushing. */
```

Comment:
> avoid empty cache after flushing

The cache will indeed be empty after flushing. Maybe say "to keep the slab partially used".

Reply:
Modified